backend/director/llm/googleai.py

```
@@ -0,0 +1,196 @@
+                "function": {
+                    "name": tool.get("name", ""),
+                    "description": tool.get("description", ""),
+                    "parameters": tool.get("parameters", {}),
+                },
+            }
+            for tool in tools
+            if tool.get("name")
+        ]
+
+    def chat_completions(
+        self, messages: list, tools: list = [], stop=None, response_format=None
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)